tasks/main.yml

```diff
@@ -5,3 +5,7 @@
 - name: Undercloud tasks
   import_tasks: undercloud.yml
   when: "'Undercloud' in group_names"
+
+- name: Overcloud tasks
+  import_tasks: overcloud.yml
+  when: "'Undercloud' not in group_names"
```

Comment:
s/Undercloud/Overcloud/

Reply:
That verifies that the group Undercloud is not in the host's current group_names. Meaning, if it's not the Undercloud, it's the overcloud.

Reply:
ok, I missed the not